src/core/configuration-component.ts

```diff
@@ -19,28 +19,37 @@ interface IConfigurable extends IConfigurationOwner {
     $_config: Configuration;
 }
 
-function initBinding(vueInstance) {
+function getConfig(vueInstance) {
     if (!vueInstance.$vnode) {
         return;
     }
 
     const componentOptions = (vueInstance.$vnode.componentOptions as any as IConfigurable);
     const config = componentOptions && componentOptions.$_config;
 
+    return config;
+}
+
+function initOptionChangedFunc(vueInstance) {
+    const config = getConfig(vueInstance);
+
     if (!config) {
         return;
     }
 
     const innerChanges = {};
     config.init(Object.keys(vueInstance.$props));
-    bindOptionWatchers(config, vueInstance, innerChanges);
     setEmitOptionChangedFunc(config, vueInstance, innerChanges);
 }
 
 const DxConfiguration: VueConstructor = Vue.extend({
 
     beforeMount() {
-        initBinding(this);
+        const innerChanges = {};
+        const config = getConfig(this) as Configuration;
+
+        initOptionChangedFunc(this);
```

Comment:
```suggestion
        initOptionChangedFunc(config);
```
since `getConfig` is going to be called twice